src/components/Explorer.js

```diff
@@ -10,18 +10,29 @@ import TimeChart from './TimeChart'
 
 import { censusData, detailData, timeData2 } from '../util/data'
 
-const Explorer = ({ params, dispatch }) => {
+const crimeIds = {
+  murder: 'murder and nonnegligent homicide',
+  rape: 'rape (legacy definition)',
+  robbery: 'robbery',
+}
+
+const Explorer = ({ appState, dispatch, params, router }) => {
   const { crime } = params
-  const state = startCase(params.state)
+  const { filters } = appState
+  const place = startCase(params.place)
```

Comment:
looks like sometimes we title case the `startCase` function and sometimes we just import it as `startcase`. i don't have a pref either way except that we pick one -- can you consistentify?